### ISSUE TITLE
API clientStatus bug

### ISSUE THREAD

**Issue by PeterSurda** [Bitmessage/PyBitmessage#1383]
**came over BM. I haven't tested it, just crosspasting**

Hmm well for me it's been working fine until just recently.

------------------------------------------------------
in --curses it never worked even be4

------------------------------------------------------
I've been running a headless BM instance for a while and I use the api.clientStatus call to monitor it.  I noticed that the networkStatus field says "notConnected" when it's connected and receiving incoming connections.
I don't know if this was today's pull or the one before that introduced it.

**Comment by g1itch** [comment]
Yes, it [looks strange](../blob/v0.6/src/api.py#L1331). Introduced in 7ed459b.
Previous edition was:
```python
if len(network.stats.connectedHostsList()) == 0:
    networkStatus = 'notConnected'
```
